Arelle/EdgarRenderer
# Edgr22.4

#### Reason for change

Provide preview of EDGAR release 22.4, planned for production on Dec 19. 2022.

Includes update of ix viewer to support PVP and CLB examples of ECD taxonomy.

#### Required Arelle version

Requires [Arelle companion pull request #492](https://github.com/Arelle/Arelle/pull/492)

#### Description of change

Add support for ecd and dev 2022a4 taxonomies.

#### Steps to Test

 * Run EDGAR draft conformance suite for release 22.4.
 * Run sample PVP and CLB instances of ECD taxonomy.
 * Confirm mustard menu handling of PVP and CLB groups (ELRs) matches ix viewer groups. 

**review**:
@Arelle/arelle
